### ISSUE TITLE
Ability to choose a custom icon

### ISSUE THREAD

**Issue by Su5eD** [Almighty-Alpaca/JetBrains-Discord-Integration#85]
Would be nice if you could choose a custom icon, for example a project logo, to show in rich presence.

**Comment by Rotinx** [comment]
Sadly not possible on a global plugin scale because all images need to be manually uploaded via the discord developers application panel which are then referenced by id.

**Comment by Almighty-Alpaca** [comment]
Yeah, this is unfortunately not possible without considerable work on the user's side.

**Comment by Almighty-Alpaca** [comment]
Gonna leave this open so others can see it too.

**Comment by derPiepmatz** [comment]
You could add a way to set a custom token and use the icons of this custom app

**Comment by Almighty-Alpaca** [comment]
This is actually already possible but requires a bit of work. While I could possibly make it easier I'm not sure if it's worth the effort considering how few people will actually benefit from it. If this gets more attention I'll think about it though

**Comment by derPiepmatz** [comment]
I for myself, don't need this feature. Just an idea that came up

**Comment by Ashark** [comment]
The project can have an svg item. See [here](https://rider-support.jetbrains.com/hc/en-us/community/posts/12111501255570-Any-way-to-change-project-icons-).
Will it be possible to use project's icon as a status icon?
